src/routes/popup/swap/index.tsx

```diff
@@ -53,9 +54,14 @@ export function SwapView() {
   const [receiveToken, setReceiveToken] = useSwapReceiveToken();
   const [showAdvanced, setShowAdvanced] = useState(false);
   const [tokenSelectorType, setTokenSelectorType] = useState<TokenSelectorType>("send");
-  const { networkFee, isLoading: isNetworkFeeLoading } = useARNetworkFee({
+  const {
+    networkFee,
+    networkFeeValue,
+    isLoading: isNetworkFeeLoading,
+  } = useARNetworkFee({
     tokenIn: sendToken.processId,
     tokenOut: receiveToken.processId,
+    doubleFee: +defiFeeDetails?.finalFeePercent > 0,
```

Comment:
@7i7o We calculate double network fee here if there is wander fee so that we can cover the network fee when we send the wander fee.

Reply:
tACK 👍